CSS/main.css

```diff
@@ -245,6 +245,7 @@ video {
 .progressbar {
   position: fixed;
   top: 0px;
+  left: 0px;

```

Comment:
This fixes a bug where the progressbar would be offset to the right when in fullscreen with a 4:3 video.